src/main/scala/io/pinecone/spark/pinecone/package.scala

```diff
@@ -15,8 +15,12 @@ package object pinecone {
       .add("namespace", StringType)
       .add("values", ArrayType(FloatType))
       .add("metadata", StringType)
-      .add("sparse_id", ArrayType(IntegerType, containsNull = true), nullable = true)
-      .add("sparse_values", ArrayType(FloatType, containsNull = true), nullable = true)
+      .add("sparse_values", StructType(
```

Comment:
This is where I'm defining the structure. Notice how it is now a new StructType (nested)

Reply:
Are the fields added to the schema nullable by default? Mainly wondering if `nullable = false` needs to be declared for the `sparse_values` struct.

Also, are we sure we need `containsNull = true` on `indices` and `values`? It seems like we wouldn't ever expect null to show up in those lists, but I may be misunderstanding.

Reply:
Discussed offline and added changes per api spec for additional fields that were not nullable